docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+# Can be enhanced with additional compose file 
+# See also https://docs.docker.com/compose/production/#modify-your-compose-file-for-production
+
+version: '3.3'
+
+services:
+  radicale:
+    image: tomsquest/docker-radicale
+    ports:
+      - "5232:5232"
+    restart: always
```

Comment:
Could we skip the restart policy ?
This is a __constraint__ for the user, as he may be already using a process manager to handle the restart policy.
From the doc:
> If restart policies don’t suit your needs, such as when processes outside Docker depend on Docker containers, you can use a process manager such as upstart, systemd, or supervisor instead.

I do prefer a minimalistic config where the user chooses the restart policy in its own docker-compose-override file. 
What do you think ?

Reply:
Indeed, this can be droped and be set somewhere else, for example an additional compose file.

Reply:
Ok, let's remove it.

Reply:
I've updated the PR yesterday. :)

Reply:
Hum, github is telling me the restart policy line is still there (I am in the train...)

Reply:
https://github.com/tomsquest/docker-radicale/pull/19/files not. :)